src/main/java/nl/booyah/iban/Main.java

```diff
@@ -4,7 +4,26 @@
 
 	public static void main(String[] args) {
 		// TODO Auto-generated method stub
-
+		
+		
+		
+	}
+	
+	public static boolean elevenTest(long aNumber){
+		
+		int lengte = String.valueOf(aNumber).length();
```

Comment:
- Not safe. E.g.: `String.valueOf(123) => 83` Guess why ;)
- Rename lengte naar length